File: SQL/Midterm files/Acme_Used_CarsWPFcs/Acme_Used_CarsWPFcs/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Acme_Used_CarsWPFcs
{
    public class Customer
    {
        private string cnumber;
        private string cname;
        private int cinventoryid;

        public Customer()
        {
            cnumber = "";
            cname = "";
            cinventoryid = 0;
        }

        public Customer(string number, string name, int inventoryid)
        {
            cnumber = number;
            cname = name;
            cinventoryid = inventoryid;
        }

        public string number
        {
            get
            {
                return cnumber;
            }
            set
            {
                cnumber = value;
            }
        }

        public string name
        {
            get
            {
                return cname;
            }
            set
            {
                cname = value;
            }
        }

        public int inventoryid
        {
            get
            {
                return cinventoryid;
            }
            set
            {
                cinventoryid = value;
            }
        }
    }
}

File: SQL/Midterm files/Acme_Used_CarsWPFvb/Acme_Used_CarsWPFvb/Customer.vb

Public Class Customer
    Private cNumber As String
    Private cName As String
    Private cInventoryID As Integer

    Sub New()
        cNumber = ""
        cName = ""
        cInventoryID = 0
    End Sub

    Sub New(number As String, name As String, inventoryid As Integer)
        cNumber = number
        cName = name
        cInventoryID = inventoryid
    End Sub

    Public Property Number As String
        Get
            Return cNumber
        End Get
        Set(value As String)
            cNumber = value
        End Set
    End Property

    Public Property Name As String
        Get
            Return cName
        End Get
        Set(value As String)
            cName = value
        End Set
    End Property

    Public Property InventoryID As Integer
        Get
            Return cInventoryID
        End Get
        Set(value As Integer)
            cInventoryID = value
        End Set
    End Property

End Class
